projects/kubernetes-sigs/image-builder/BOTTLEROCKET_RELEASES

```diff
@@ -1,14 +1,14 @@
 1-20:
-    ova-release-version: v1.10.1
+    ova-release-version: v0.10.1
```

Comment:
This looks wrong?

Reply:
Yeah i was keeping the version for 1.20 because BR deprecated 1.20 looks like but then we would need different versions of the admin container image for the different BR versions since we wont be deprecating 1.20 till the December minor release.